frame/support/src/traits/hooks.rs

```diff
@@ -359,6 +359,13 @@ pub trait Hooks<BlockNumber> {
 	fn integrity_test() {}
 }
 
+/// A trait to define the build function of a runtime genesis config.
+pub trait RuntimeGenesisBuild: Default + sp_runtime::traits::MaybeSerializeDeserialize {
```

Comment:
what is the reason for bounding Default and MaybeSerializeDeserialize ?

Reply:
Good question, I've blindly copied them from `GenesisBuild` for pallet.

Theoretically both of them could be removed here, take a look on the usage: 
https://github.com/paritytech/substrate/blob/1fd76624813936292a6542d4b033baa5c70e06bf/client/genesis-builder/src/lib.rs#L39-L52